common/plugins/org.jboss.tools.common.model/src/org/jboss/tools/common/model/plugin/ModelPlugin.java

```diff
@@ -103,11 +114,20 @@ public void stop(BundleContext context) throws Exception {
 	}
 	
 	private void cleanTempFiles() {
-		if(tempFolder != null && tempFolder.exists()) {
-			File[] fs = tempFolder.listFiles();
+		cleanEFS_Files(tempFolder, System.currentTimeMillis() - 30l * 24l * 60l * 60l * 1000l); //in global folder remove only files one month old.

```

Comment:
Not sure we actually need any timeout period for temp files. If we move this files from the shared system temp folder to the eclipse workspace then the only one eclipse instance is supposed to work with this worskspace/temp folder at the time. Do we need to store temp files from previous Eclipse sessions?